_studio/mfx_lib/encode_hw/hevc/linux/g12/hevcehw_g12_caps_lin.h

```diff
@@ -51,8 +51,8 @@ class Caps
         caps.NoInterlacedField          = 1;
```

Comment:
for me this struct with hardcoded caps is a surprise. We have an interface for caps querying. @eistarov , @mgonchar , @dvrogozh , why do we need to hardcode caps?

Reply:
By our agreement with HEVC scrum, we aligned refactored code logic with legacy one by the moment of ww41, all changes made after (if any) supposed to be replicated in refactored code by HEVC scrum. I don't track status of TGL caps readiness in driver.

Reply:
I heard that there were some issues with caps reported by driver. @saosipov may say more here, but he is on the vacation now